automol/util/dict_/_dict_.py

```diff
@@ -107,56 +115,58 @@ def values_in_multilevel_dct(dct, key1, key2, fill_val=None):
     return val
 
 
-def keys_by_value(dct, func=lambda x: x):
-    """return dictionary keys for specific values"""
+def keys_by_value(dct: dict[object, object], func: Callable = lambda x: x):
+    """Return dictionary keys for specific values."""
     return frozenset(key for key, val in dct.items() if func(val))
 
 
-def transform_keys(dct, func=lambda x: x):
-    """apply a function to each key"""
-    return dict(zip(map(func, dct.keys()), dct.values()))
+def transform_keys(dct: dict[object, object], func: Callable = lambda x: x):
+    """Apply a function to each key."""
+    return dict[object, object](zip(map(func, dct.keys()), dct.values(), strict=False))
 
 
-def transform_values(dct, func=lambda x: x):
-    """apply a function to each value"""
-    return dict(zip(dct.keys(), map(func, dct.values())))
+def transform_values(dct: dict[object, object], func: Callable = lambda x: x):
+    """Apply a function to each value."""
+    return dict[object, object](zip(dct.keys(), map(func, dct.values()), strict=False))
```

Comment:
Take out the `[object, object]` on this line. We only want that for the type, not for the actual conversion.